src/contexts/SignedInUserContext.jsx

```diff
@@ -64,7 +64,9 @@ export const SignedInUserContextProvider = ({ children }) => {
         setProfileData(fetchedProfileData);
         await Promise.all([
           createPASSContainer(session, fetchedPodUrl, 'Documents'),
-          createPASSContainer(session, fetchedPodUrl, 'Profile')
+          createPASSContainer(session, fetchedPodUrl, 'Profile'),
+          createPASSContainer(session, podUrl, 'Outbox'),
```

Comment:
You shouldn't need to create these containers in advance. You should be able to create them in the hook the first time the user tries to access the record.

See how we handle that in the `useRdfCollection` here: https://github.com/codeforpdx/PASS/blob/9b578d7a1ff96aa1f0eeb332e76e2997bc74d9e4/src/hooks/useRdfCollection.js#L51

Reply:
I think I could remove the one for Outbox since it's internally for the user so users should still be able to access them without needing to generate them.

For the Inbox, I'll try switching up the function to just generate the necessary `.acl` file without the need for making the container so others could send message to them. Without it, could only send messages to themselves.

Reply:
Alright, think I've managed to get this working. I've got the new generateACL function nested inside the `fetchDocument` function for `useMessageList`. It'll only trigger when logging in for the first time right after the `saveSolidDatasetAt` function.